pages/learn/develop/container-contracts.md

```diff
@@ -1,19 +1,19 @@
 ---
-title: Container Contracts
+title: Container contracts
 excerpt: A guide to using container contracts to enforce device compatibility
 ---
 
 # Container contracts
 
 __Note__ Container contracts are available for devices running Supervisor versions >= 10.6.0. All prior versions will not enforce any contracts.
 
-Container contracts are used to ensure the compatibility of a device to run an individual service of an application. For example, container contracts may be used to ensure that the host operating system is running a specific version of balenaOS or balena-supervisor such that if an application is dependent on a specific version, it only gets deployed to those devices that can meet these requirements to fulfill the contract. Currently, you may define contract requirements based on the versions of [balenaOS](/reference/OS/overview/2.x/) and [balena-supervisor](/reference/supervisor/supervisor-api/).
+Container contracts are used to ensure the compatibility of a device to run an individual service of an application. For example, container contracts may be used to ensure that the host operating system is running a specific version of [{{ $names.os.lower }}][os] or [{{ $names.company.lower }} Supervisor][supervisor] such that if an application is dependent on a specific version, it only gets deployed to those devices that can meet these requirements to fulfill the contract. Currently, you may define contract requirements based on the versions of {{ $names.os.lower }} and Supervisor.
```

Comment:
I assume on devices that don't meet the requirements will still download the images? or does it not get deployed at all. I thought the contract only enforced if it was run or not, but all of them would have it?

Reply:
It won't be downloaded in the contract fails - this is something that we did discuss and I'd like to implement, but I'd rather it was a future work thing.

Reply:
Just to be clear, I think this is fully detailed later. Needs more clarification here?